### ISSUE TITLE
Failed to parse source map from...

### ISSUE THREAD

**Issue by freynolds-plannery** [tcampb/react-calendly#169]
In last version of react-scripts, warnings are now being shown for problems with parsing source maps in dependencies. As a result, with every run of react-scripts, there is a warning:

```
WARNING in ./node_modules/react-calendly/dist/index.es.js
Module Warning (from ./node_modules/react-scripts/node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '.../node_modules/react-calendly/src/components/PopupModal/Modal.tsx' file: Error: ENOENT: no such file or directory, open '.../node_modules/react-calendly/src/components/PopupModal/Modal.tsx'
```

https://github.com/facebook/create-react-app/pull/11752

From what I've seen in other repos, adding the source map to the package should solve the issue

**Comment by freynolds-plannery** [comment]
Example: https://github.com/demvsystems/yup-ast/pull/24/files

**Comment by tcampb** [comment]
Thanks for creating this issue @freynolds-plannery! This issue should be fixed in version `4.2.1`.

**Comment by freynolds-plannery** [comment]
you rock!
